dvc/command/repro.py

```diff
@@ -150,7 +155,19 @@ def add_arguments(repro_parser):
         "--glob",
         action="store_true",
         default=False,
-        help="Allows targets containing shell-style wildcards.",
+        help=(
+            "Allows targets containing shell-style wildcards to match out "
+            "files.",
```

Comment:
comma inside the parentheses turn into a tuple. This is making `dvc repro -h` fail.

Reply:
Same issue in the `glob-stages` too.